pkg/i18n/english.go

```diff
@@ -795,6 +796,7 @@ func EnglishTranslationSet() TranslationSet {
 		NoChangedFiles:                      "No changed files",
 		PullWait:                            "Pulling...",
 		PushWait:                            "Pushing...",
+		RebaseWait:                          "Rebasing...",
```

Comment:
The fields ending in 'Wait' are for popup panels, but for rebases I don't want a popup. Instead we want to add a new field under the `Actions` field for this and have the value be `Rebasing` (no `...`)